BTCPayServer.Tests/docker-compose.yml

```
@@ -323,4 +323,4 @@ networks:
     driver: bridge
     ipam:
       config:
-        - subnet: 172.23.0.0/16
+        - subnet: 172.23.0.0/16
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)